packages/velo-external-db-core/test/drivers/filter_transformer_test_support.js

```diff
@@ -20,7 +20,7 @@ const givenFilterByIdWith = (id, filter) => {
                                 })
 }
 
-const stubIgnoreTransform = (filter) => {
+const givenTransformResult = (filter) => {
     when(filterTransformer.transform).calledWith(filter)
                                 .mockReturnValue(filter)
```

Comment:
transform needs to return another value, the contract is that filter changes so let's do that here too.